src/Components/Forms/CuratorPicker.php

```diff
@@ -352,6 +352,9 @@ public function getViewAction(): Action
             ->label(trans('curator::views.picker.view'))
             ->icon('heroicon-s-eye')
             ->color('gray')
+            ->visible(function () {
```

Comment:
Is there really a use case where a user would be able to create/edit a record but not be able to manage it's media?